src/SubmitPage/SubmitForm.js

```diff
@@ -141,7 +141,7 @@ class SubmitForm extends React.Component {
   onChangeInput({ target }, dataKey) {
     this.setState({
       isValid: true,
-      data: { ...this.state.data, [dataKey]: target.value },
+      data: { ...this.state.data, [dataKey]: titleCase(target.value) },
```

Comment:
I think we should just do this in the view part instead during submission. This will ensure that all the existing entries are formatted without changing anything in the DB. what do you think ?

Reply:
also. this `onChangeInput` is handling inputs for all inputs I think. we only want to apply title case on Store Name

Reply:
Nice caught, missed that thnx

Reply:
Change is DB is suggested in the issue itself, I do agree with that all the changes should be explicit to user and one formatted at source you do not need reformatting at any other location.

Reply:
Let's just put this on view for now. We can handle DB migration and formatting during submission in another ticket.

Reply:
Another reason i want to avoid this formatting during submission is because the store information can be both submitted and updated. I'm not sure how backend will behave if store name is updated in an existing entry

Reply:
what's the reason to doubt update flow, Updated entry if saved will update the backend as well. It should be just a case of rather supplying raw input to backend we supply formatted input. 

Correct me if that is not the case

Reply:
Yeah. I just checked, existing stores are searched by `placeId`. so store name should ideally not affect much